Added Random Anime Profile pic generator Project

AmolShelke2 opened the pull request:
Hey, Project Admin I've Added a Random anime profile pic generator project to Ace-The-FrontEnd Project, it is created using HTML CSS and vanilla javaScript, It is a very helpful project for beginner developers who have don't knowledge of web API using the third-party API how to fetch the data using fetch function in javaScript I hope that you will go through my pull request and merge it..

![192 168 187 73_5501_Random%20Anime%20Profile%20Pic_index html](https://user-images.githubusercontent.com/95171638/156553845-5032ae41-7a8f-43d9-bd48-c4905d9ee15a.png)

surajm-333 commented:
@AmolShelke2 remove .vscode folder and remove all the changes from repository's readme file

AmolShelke2 commented:
> @AmolShelke2 remove .vscode folder and remove all the changes from repository's readme file

I've removed the .vscode folder and removed all the changes from the repository's README.md file

surajm-333 commented:
@AmolShelke2 add a gif depicting working of this project in the project folder.

AmolShelke2 commented:
> @AmolShelke2 add a gif depicting working of this project in the project folder.

yes sure

surajm-333 commented:
Nice work @AmolShelke2. From next time mention the issue number in pr template.

anamansari062 commented:
@AmolShelke2 This pr has already been made in this repo https://github.com/ZeroOctave/ZeroOctave-Javascript-Projects/pull/310 . This things can get you banned. I will be removing the levels
